### PR TITLE
Fill "Cite Us" section

### PR DESCRIPTION
**Pull request by efaulhaber**

### CODE REVIEW

**Comment by svchb** [comment]
You forgot the citation.bib file?

**Review by svchb — DISMISSED** [review]

**Review by LasNikas — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 [![codecov](https://codecov.io/github/trixi-framework/TrixiParticles.jl/branch/main/graph/badge.svg?token=RDZXYbij0b)](https://codecov.io/github/trixi-framework/TrixiParticles.jl)
 [![SciML Code Style](https://img.shields.io/static/v1?label=code%20style&message=SciML&color=9558b2&labelColor=389826)](https://github.com/SciML/SciMLStyle)
 [![License: MIT](https://img.shields.io/badge/License-MIT-success.svg)](https://opensource.org/licenses/MIT)
+[![DOI](https://zenodo.org/badge/505448384.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
```
Comment (sloede) [inline README.md:11]:
```suggestion
[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10797541.svg)](https://zenodo.org/doi/10.5281/zenodo.10797541)
```
Add proper badge number

**Review by sloede — CHANGES_REQUESTED** [review]
LGTM once the proper badge number is used. Thanks for updating this!

File: README.md (hunk 1 above)
Comment (efaulhaber) [reply README.md:11]: Interesting. Why is this wrong? I just copied it from Zenodo:
grafik

Comment (sloede) [reply README.md]: AFAIU, ...42 is the specific, current release, while ...41 is the super DOI that always points to the latest release. The badge link you used resolved to ..42 for me. I strongly suggest to always use the ...41 DOI whenever you reference TrixiP

**Review by sloede — APPROVED** [review]

**Pull request merged**